cmd/statusd/main.go

```diff
@@ -63,7 +63,7 @@ var (
 		params.FleetProd,
 		fmt.Sprintf(
 			"Select fleet: %s (default %s)",
-			[]string{params.FleetProd, params.FleetStaging, params.FleetTest, params.FleetWakuV2Prod, params.FleetWakuV2Test}, params.FleetProd,
+			[]string{params.FleetProd, params.FleetStaging, params.FleetTest, params.FleetWakuV2Prod, params.FleetWakuV2Test, params.FleetStatusTest}, params.FleetProd,
```

Comment:
This line is getting silly. Maybe we should make this a separate multi-line declaration of a variable?